k8s/namespaces/sscs/sscs-evidence-share/perftest.yaml

```diff
@@ -4,18 +4,8 @@ metadata:
   name: sscs-evidence-share
   namespace: sscs
 spec:
-  releaseName: sscs-evidence-share
-  rollback:
-    enable: true
-    retry: true
-  chart:
-    git: git@github.com:hmcts/hmcts-charts
-    ref: master
-    path: stable/sscs-evidence-share
   values:
     java:
-      replicas: 2
-      useInterpodAntiAffinity: true
       image: hmctspublic.azurecr.io/sscs/evidence-share:prod-bb91c3b-20220510083202 #{"$imagepolicy": "flux-system:sscs-evidence-share"}
```

Comment:
(Minor) the prod image can also be moved to the base as most environments use it.

Reply:
Image added to base chart